simulation_parameters/common/easter_egg_messages.json

```diff
@@ -0,0 +1,35 @@
+{
+    "Messages": [
+        "Fun Fact, The Didinium  and Paramecium are a textbook example of a predator prey relationship that has been studied for decades, now are you the Didinium, or the Paramecium? Predator, or Prey?",
+        "Here's a tip, toxins can be used to knock other toxins away from you if you are quick enough.",
+        "Here's a tip, Osmoregulation costs 1 ATP per second per hex your cell has, each empty hex of cytoplasm generates 5 ATP per second aswell, which means if you are losing ATP due to osmoregulation just add a couple empty hexes cytoplasm or remove some organelles.",
+        "Fun Fact, In real life prokaryotes have something called Biocompartments which act like organelles, and are in fact called Polyhedral organelles",
+        "Fun Fact, The metabolosome is what is called a Polyhedral organelle",
+        "Here's a Tip, Chromatophores generate 1/3rd the glucose of a chloroplast",
+        "Here's a Tip, You generate exactly 2 glucose per second per chemoplast, as long as you have at least 1 hydrogen sulfide to convert.",
```

Comment:
might be inaccurate

Reply:
Should I remove those? (or add "these tips are inaccurate warnings" somewhere)

Reply:
just remove them